ntoskrnl/mm/section.c

```diff
@@ -3700,7 +3701,9 @@ MiRosUnmapViewOfSection(
             }
         }
         DPRINT("One mapping less for %p\n", ImageSectionObject->FileObject->SectionObjectPointer);
-        InterlockedDecrement(&ImageSectionObject->MapCount);
+        MapCount = InterlockedDecrement(&ImageSectionObject->MapCount);
```

Comment:
NIT: you can probably get away with:
`ULONG MapCount = InterlockedDecrement(&ImageSectionObject->MapCount);`